Resources/Azure/ArmTemplates/template.json

```diff
@@ -722,6 +722,14 @@
               {
                 "name": "CosmosDbSettings__ConnectionString",
                 "value": "[parameters('providerPortalCosmosDbConnectionString')]"
+              },
+              {
```

Comment:
Suggest adding a prefix to these like the others so we can easily bind an `IOptions<T>` from a config section later if required

Reply:
I think we should cross that bridge when we come to it as these values aren't exposed any further than this application at this time.